packages/botonic-react/src/components/multichannel/multichannel.jsx

```diff
@@ -22,7 +24,7 @@ export const Multichannel = props => {
   }
   if (isFacebook(requestContext.session)) {
     const newChildren = deepMapWithIndex(props.children, child => {
-      if (child && child.type && child.type.name === COMPONENT_TYPE.TEXT) {
+      if (child && child.type && child.type.name === Text.name) {
```

Comment:
Refactor: You could change this (and all the other conditions below) to:
```suggestion
      if (isText(child)) {
```
This `isText` function doesn't exist now, but should be the same as `isPic`, `isButton`, etc.

Reply:
applied